src/OpenGLES/EAGLContext.cs

```diff
@@ -15,6 +20,29 @@ public EAGLContext ()
 		{
 		}
 #endif
+
+		[iOS (10,0)]
+		[TV (10,0)]
+		public bool PresentRenderBuffer (nuint target, double presentationTime)
+		{
+			return _PresentRenderbufferAtTime (target, presentationTime);
+		}
+
+		[iOS (10,3)]
+		[TV (10,2)]
+		public bool PresentRenderBuffer (nuint target, double presentationTime, PresentationMode mode)
+		{
+			switch (mode) {
+			case PresentationMode.AtTime:
+				return _PresentRenderbufferAtTime (target, presentationTime);
+				break;
```

Comment:
no need for `break` after a `return` statement. IIRC the C# compiler should be warning you about this...